hilti/toolchain/include/ast/types/bitfield.h

```diff
@@ -133,6 +135,9 @@ class Bitfield : public hilti::TypeBase,
     auto typeParameters() const { return hilti::util::slice(children(), 2); }
     /** Implements the `Type` interface. */
     auto isWildcard() const { return _wildcard; }
+
+    std::optional<uint64_t> size() const { return width() / 8; }
+
```

Comment:
```suggestion
     /** Implements the `Type` interface. */
    std::optional<uint64_t> size() const { return width() / 8; }
```
(just for consistency)